lib/teslamate/vehicles/vehicle.ex

```diff
@@ -401,10 +418,10 @@ defmodule TeslaMate.Vehicles.Vehicle do
 
         interval =
           case state do
-            {:driving, _, _} -> 10
-            {:charging, _} -> 15
-            :online -> 20
-            _ -> 30
+            {:driving, _, _} -> round(default_interval() * 2 / 3)
+            {:charging, _} -> default_interval()
+            :online -> round(default_interval() * 4 / 3)
+            _ -> default_interval() * 2
```

Comment:
My eyes immediately get drawn to these hardcoded factors - what do they do?

Reply:
Initially, I preferred to keep this factor rather than the current numerical values. Current values ​​are 10, 15, 20, 30.

It's only in the case of unhandled errors that these values ​​are used, so I hesitated to leave the current values ​​hard…

As only the 200 response code are important for the Tesla WAF, I think now it’s better to revert to current hardcoded values.

Reply:
Reverted to default values.